autoarray/inversion/inversions.py

```diff
@@ -227,9 +206,50 @@ def brightest_reconstruction_pixel(self):
     @property
     def brightest_reconstruction_pixel_centre(self):
         return grids.GridIrregular(
-            grid=[self.mapper.pixelization_grid[self.brightest_reconstruction_pixel]]
+            grid=[
+                self.mapper.source_pixelization_grid[
+                    self.brightest_reconstruction_pixel
+                ]
+            ]
         )
 
+    @property
+    def mapped_reconstructed_image(self):
+        raise NotImplementedError()
+
+    @property
+    def residual_map(self):

```

Comment:
Prefer using ABC and abstractmethod to NotImplementedError

Reply:
Will do!